### DISCUSSION TITLE
카테고리 옆에 원래 NEW아이콘 안뜨나요?

### DISCUSSION THREAD

**Discussion by bank365** [pronist/hELLO#233] [Q&A] [answered]
안녕하세요
방금 포스트 햇는데
카테고리 옆에  NEW아이콘이 표기가 안되어 있네요
원래 그런건가요?
원래 그런거라면 NEW마크 뜨게 수정해서 배포해주시면 안될가요?ㅠ

**Comment by pronist** [comment, marked as answer]
@bank365 네 안 나옵니다. 스킨에서 디자인상 배제하였습니다.

**Comment by zineeworld** [comment]
@bank365 CSS 수정하시면 됩니다.

#category .tt_category .link_tit img, #category .tt_category .link_item img, #category .tt_category .link_sub_item img {
    display: inline-block;
}
